src/overlays/effects/ovl_Effect_Ss_Extra/z_eff_ss_extra.c

```diff
@@ -1,29 +1,97 @@
 /*
  * File: z_eff_ss_extra.c
  * Overlay: ovl_Effect_Ss_Extra
- * Description:
+ * Description: The floating points hovering over a target you hit in OoT
+ *   i.e. 30/60/100 points when hitting the target in lost woods with slingshot to get a bigger seed sack
```

Comment:
Regarding these points being used in swamp archery, the description can use an update